windows/vbs_enable_disable/vbs_enable_disable.yml

```diff
@@ -37,6 +37,18 @@
               {%- else -%}Others
               {%- endif -%}
 
+        - name: "Skip test case"
+          include_tasks: ../../common/skip_test_case.yml
+          vars:
+            skip_msg: >-
+              Skip test case due to using Intel CPUs for VBS requires vSphere 6.7 or later and using AMD CPUs for VBS
+              requires vSphere 7.0 Update 2 or later. 
+              Please refer to https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-security/GUID-CE292D3F-D4AC-4607-B262-DE19CE6E9F6B.html
+          skip_reason: "Not Supported"
+          when: >-
+            (vm_hardware_version_num | int < 14 and esxi_cpu_model == "Intel") or
```

Comment:
Qi added a parameter `esxi_cpu_vendor` in merge request #554

Reply:
esxi_cpu_model_info is coming from "esxi_host_property_results.ansible_facts.summary.hardware.cpuModel", while esxi_cpu_vendor is coming from esxi_host_property_results.ansible_facts.hardware.cpuPkg[0].vendor. 

I think for this scenario, cpuModel is more precise.

Reply:
If that info is already defined, then here you no need to define it again:
```
 - name: "Set ESXi CPU model"
          ansible.builtin.set_fact:
            esxi_cpu_model: |-
              {%- if 'AMD' in esxi_cpu_model_info -%}AMD
              {%- elif 'Intel' in esxi_cpu_model_info -%}Intel
              {%- else -%}Others
              {%- endif -%}
```
if you find that value is not accurate, then we need to change it instead of defining a new one.

Reply:
Here does not create a new one, just use the value of esxi_cpu_model_info got in esxi_get_model.yml. 
When above code was added for VBS, there is only variable esxi_cpu_model_info, no esxi_cpu_vendor.  esxi_cpu_vendor was added later, not sure why https://github.com/vmware/ansible-vsphere-gos-validation/pull/554 does not use the already existing one.

Reply:
As we talked offline, will use esxi_cpu_vendor here.